applications/rag/README.md

```diff
@@ -56,9 +56,12 @@ Next, set up the inference server, the `pgvector` instance, Jupyterhub, Kuberay
 
 1. `cd ai-on-gke/applications/rag`
 
+2. (Optional) [Enable IAP and Set up Brand](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/modules/jupyter/authentication/README.MD#enable_iap_service) (Recommend)
+
 2. Edit `workloads.tfvars` with your project ID, cluster name, location and a GCS bucket name.
     * The GCS bucket name needs to be globally unique so add some random suffix to it (ensure `gcloud storage buckets describe gs://<bucketname>` returns a 404).
     * Optionally choose the k8s namespace & service account to be used by the application. If not selected, these resources will be created based on the default values set.
+    * Optionally, set add_auth to be true to create load balancer and IAP
```

Comment:
In a separate bullet let's explain the other parameters, such as allowlisted members, support email, etc.